<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Sonali-PC" failures="1" tests="7" name="com.lean.qa.TestCases.primefacesLoginTest" time="104.808" errors="0" timestamp="23 Jan 2020 20:02:11 GMT" skipped="0">
  <testcase classname="com.lean.qa.TestCases.primefacesLoginTest" name="verifyLoginLogoDisplay" time="14.531"/>
  <testcase classname="com.lean.qa.TestCases.primefacesLoginTest" name="verifyUsernameDisplay" time="16.673"/>
  <testcase classname="com.lean.qa.TestCases.primefacesLoginTest" name="verifyPasswordDisplay" time="13.884"/>
  <testcase classname="com.lean.qa.TestCases.primefacesLoginTest" name="loginTest" time="13.116"/>
  <testcase classname="com.lean.qa.TestCases.primefacesLoginTest" name="loginTest1" time="15.236"/>
  <testcase classname="com.lean.qa.TestCases.primefacesLoginTest" name="loginTest3" time="11.885"/>
  <testcase classname="com.lean.qa.TestCases.primefacesLoginTest" name="verifyloginTest" time="19.483">
    <failure message="expected [] but found [admin]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [admin]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.lean.qa.TestCases.primefacesLoginTest.verifyloginTest(primefacesLoginTest.java:81)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyloginTest -->
</testsuite> <!-- com.lean.qa.TestCases.primefacesLoginTest -->
